### PR TITLE
Update to new Microsoft.AspNetCore.Server.IntegrationTesting

### PR DESCRIPTION
**Pull request by ryanbrandenburg**

### CODE REVIEW

File: build/dependencies.props [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <Project>
   <PropertyGroup>
     <AspNetCoreVersion>2.0.0-*</AspNetCoreVersion>
-    <AspNetCoreLabsVersion>0.3.0-*</AspNetCoreLabsVersion>
+    <AspNetCoreLabsVersion>0.4.0-*</AspNetCoreLabsVersion>
```
Comment (pranavkm) [inline build/dependencies.props]: Rename this to `AspNetIntegrationTestingVersion`

**Comment by ryanbrandenburg** [comment]
Updated, good idea on the rename @pranavkm.

**Comment by mikeharder** [comment]
If this fixes tests, let's merge ASAP and react to feedback later.

**Comment by anurse** [comment]
:shipit:

**Pull request merged**